lib/travis/api/v3/routes.rb

```diff
@@ -22,6 +22,10 @@ module Routes
       end
     end
 
+    resource :builds do
```

Comment:
Hmmmm. You are right. This is what @rkh recommended to me, but lets go ahead with what you have and I can check with him when he is back. Thanks for pointing this out.